exercises/high-scores/example.py

```diff
@@ -0,0 +1,25 @@
+class HighScores(object):
+    def __init__(self, scores):
+        self.scores = scores
+
+    @property
```

Comment:
I think traditional methods would be more clear than properties in this instance.